lib/Backend/amd64/LowererMDArch.cpp

```diff
@@ -523,7 +561,9 @@ LowererMDArch::LowerCallArgs(IR::Instr *callInstr, ushort callFlags, Js::ArgSlot
         callInstr->InsertBefore(argInstr);
         argCount++;
     }
-
+#ifndef _WIN32
+    this->xplatCallArgs.StopRecording();
```

Comment:
This is a pretty neat idea 👍 I'm worried about the locations of `Start/Stop` recording though. It seems to me maybe every `GetArgSlotOpnd` call needs to be recorded. We use a mixture of LowerCallArgs and several other random code paths to load different args, which may all need to be homed. E.g., even the `helperCallArgs` loaded in `LowerCall` may need to be homed. Stopping here may miss some args. Please survey all calls to `GetArgSlotOpnd` to see if this is a potential issue.